src/App.jsx

```diff
@@ -0,0 +1,17 @@
+import Login from './components/Login';
+import { Route, Routes } from 'react-router-dom';
+
+function App() {
+
+  return (
+    <div className='bg-gradient-to-br from-custom-gradient-0 via-custom-gradient-1 to-custom-gradient-2 flex items-center justify-center h-screen' >
```

Comment:
Pra ficar mais fácil de ler o tailwind, seria uma boa usar nomes de cores para o seu gradient. Por exemplo `from-yellow-primary via-pink-primary to-blue-primary`. Ao invés de primary, pode ser algo que descreva o tom da cor.